x-pack/filebeat/module/googlecloud/firewall/test/rare.log-expected.json

```diff
@@ -0,0 +1,120 @@
+[
+    {
+        "@timestamp": "2019-11-06T16:41:38.394Z",
+        "destination.address": "10.128.0.16",
+        "destination.domain": "local-adrian-test",
+        "destination.ip": "10.128.0.16",
+        "destination.port": 80,
+        "event.category": "firewall-rule",
```

Comment:
`event.category` is still reserved, and at this time, there is no plan to have a category named "firewall-rule".

Leaving this here signs you up to have to do a breaking change, once the expected values are published for `event.category`. The name of the category that will encapsulate firewall rule events is still very much in flux.

cc @MikePaquette